packages/e2e/cypress/fixtures/Auth.currentAuthenticatedUser-setup-TOTP.json

```diff
@@ -0,0 +1,3 @@
+{
+  "email": "test@example.com"
```

Comment:
In the real `Auth.currentAuthenticatedUser` callback there are many different values and methods. For simplicity sake, all I care is that this is being called and it returns this value for email.

Reply:
*changed this to username instead.